incubator/concourse/values.yaml

```diff
@@ -0,0 +1,375 @@
+  authDuration: 24h
+
+  ## Interval on which to check for new versions of resources.
+  ## Use Go duration format (1m = 1 minute).
+  ##
+  resourceCheckingInterval: 1m
+
+  ## How long to cache the result of a get step after a newer version of the resource is found.
+  ## Use Go duration format (1m = 1 minute).
+  ##
+  oldResourceGracePeriod: 5m
+
+  ## The interval on which to check for and release old caches of resource versions.
+  ## Use Go duration format (1m = 1 minute),
+  ##
+  resourceCacheCleanupInterval: 30s
+
+  ## URL used to reach any ATC from the outside world.
+  ##
+  # externalURL:
+
+  ## An URL pointing to the Docker registry to use to fetch Docker images.
+  ## If unset, this will default to the Docker default
+  ##
+  # dockerRegistry:
+
+  ## Docker registry(ies) (comma separated) to allow connecting to even if not secure.
+  ##
+  # insecureDockerRegistry:
+
+  ## Application client ID for enabling GitHub OAuth.
+  ##
+  # githubAuthClientId:
+
+  ## Application client secret for enabling GitHub OAuth.
+  ##
+  # githubAuthClientSecret:
+
+  ## GitHub organizations (comma separated) whose members will have access.
+  ##
+  # githubAuthOrganization:
+
+  ## GitHub teams (comma separated) whose members will have access.
+  ##
+  # githubAuthTeam:
+
+  ## GitHub users (comma separated) to permit access.
+  ##
+  # githubAuthUser:
+
+  ## Override default endpoint AuthURL for Github Enterprise.
+  ##
+  # githubAuthAuthUrl:
+
+  ## Override default endpoint TokenURL for Github Enterprise.
+  ##
+  # githubAuthTokenUrl:
+
+  ## Override default API endpoint URL for Github Enterprise.
+  ##
+  # githubAuthApiUrl:
+
+  ## Name for this auth method on the web UI.
+  ##
+  # genericOauthDisplayName:
+
+  ## Application client ID for enabling generic OAuth.
+  ##
+  # genericOauthClientId:
+
+  ## Application client secret for enabling generic OAuth.
+  ##
+  # genericOauthClientSecret:
+
+  ## Generic OAuth provider AuthURL endpoint.
+  ##
+  # genericOauthAuthUrl:
+
+  ## Parameters (comma separated) to pass to the authentication server AuthURL.
+  ##
+  # genericOauthAuthUrlParam:
+
+  ## Optional scope required to authorize user.
+  ##
+  # genericOauthScope:
+
+  ## Generic OAuth provider TokenURL endpoint.
+  ##
+  # genericOauthTokenUrl:
+
+## Configuration values for Concourse Web components.
+##
+web:
+  ## Override the components name (defaults to web).
+  ##
+  # nameOverride:
+
+  ## Number of replicas.
+  ##
+  replicas: 1
+
+  ## Configure resource requests and limits.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    requests:
+      cpu: "100m"
+      memory: "128Mi"
+
+  ## Service configuration.
+  ## ref: https://kubernetes.io/docs/user-guide/services/
+  ##
+  service:
+    ## For minikube, set this to NodePort, elsewhere use LoadBalancer.
+    ## ref: https://kubernetes.io/docs/user-guide/services/#publishing-services---service-types
+    ##
+    type: NodePort
```

Comment:
Make this a ClusterIP service and use port forwarding by default since the out of the box config uses static credentials.